typings/index.d.ts

```diff
@@ -952,6 +952,7 @@ declare module 'discord.js' {
     public equals(message: Message, rawData: object): boolean;
     public fetchWebhook(): Promise<Webhook>;
     public fetch(): Promise<Message>;
+    public crosspost(): Promise<Message>;
```

Comment:
```suggestion
    public crosspost(): Promise<this>;
```
i think this would make more sense right?

Reply:
That'd be very inconsistent with all the other typings.

Reply:
ah, ok